core/federated/RTI/enclave.c

```diff
@@ -0,0 +1,291 @@
+#include "enclave.h"
+
+/**
+ * Reference to enclave_RTI_t instance.
+ */
+enclave_RTI_t* _E_RTI;
+
+// Global variables defined in tag.c:
+extern instant_t start_time;
+
+// RTI mutex, which is the main lock  
+extern lf_mutex_t rti_mutex;
+
+// FIXME: For log and debug message in this file, what sould be kept: 'enclave', 
```

Comment:
This seems like something that is very actively worked on, so it might be OK to leave these "as is." Though, if there is a chance that these won't be addressed in a matter of days or weeks, please create an issue and reference it in the code.

Reply:
This should be addressed in the upcoming weeks (within the design of enclaves and federates...).
So let it here for now.